specification/containerinstance/resource-manager/Microsoft.ContainerInstance/preview/2018-02-01-preview/containerInstance.json

```diff
@@ -314,6 +321,39 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.ContainerInstance/locations/{location}/operations/{operationId}": {
+      "get": {
+        "operationId": "AsyncContainerGroupOperation_Get",
```

Comment:
Name seems unrelated to semantic of the operation.

Reply:
Unless there is something I don't get in your service, if this is the operation to follow your previous async operation, you don't need this at all. If you follow ARM guidelines, this operation will be discovered by the "x-ms-long-running" flag.

Reply:
There is not need to add this to the swagger spec.